kubectl-plugin/install/kind.go

```diff
@@ -123,106 +146,100 @@ func (o *kindOption) runE(cmd *cobra.Command, args []string) (err error) {
 	return
 }
 
-func loadImagesOfDevOps() (err error) {
+func (o *kindOption) loadImagesOfDevOps() (err error) {
 	var wg sync.WaitGroup
-	if err = pullAndLoadImageSync("kubesphere/jenkins-uc:v3.0.0", &wg); err != nil {
-		return
-	}
-	if err = pullAndLoadImageSync("jenkins/jenkins:2.176.2", &wg); err != nil {
-		return
-	}
-	if err = pullAndLoadImageSync("jenkins/jnlp-slave:3.27-1", &wg); err != nil {
-		return
-	}
-	if err = pullAndLoadImageSync("kubesphere/builder-base:v2.1.0", &wg); err != nil {
-		return
-	}
-	if err = pullAndLoadImageSync("kubesphere/builder-nodejs:v2.1.0", &wg); err != nil {
-		return
-	}
-	if err = pullAndLoadImageSync("kubesphere/builder-go:v2.1.0", &wg); err != nil {
-		return
+
+	images := map[string]string{
+		"kubesphere/jenkins-uc":     "v3.0.0",
```

Comment:
IMO, some of them are deprecated and should not be used. Or we can determine the tags according to version of KubeSphere.

Reply:
But the issue is unrelated with the PR. So this won't stop another maintainers from merging this though.